### PR TITLE
EmojiParameter refactor

### PR DESCRIPTION
**Pull request by davipatury**
### TASKLIST
- [x] REFACTOR SHITCODE

### CODE REVIEW

File: src/structures/command/parameters/types/EmojiParameter.js [hunk 1]
```diff
@@ -12,31 +11,14 @@ module.exports = class EmojiParameter extends Parameter {
...
-    let isAnimated = ''
-
+  static parse (arg, { t, client, guild }) {
+    const regexResult = EMOJI_REGEX.exec(arg)
     if (regexResult) {
-      if (this.sameGuildOnly && !guild.emojis.get(regexResult[2])) throw new CommandError(t('errors:emojiNotFromSameGuild'))
-      name = regexResult[1]
-      id = regexResult[2]
-      isAnimated = Boolean(animatedRegexResult)
+      const [ ,,, id ] = regexResult
```
Comment (davipatury) [inline src/structures/command/parameters/types/EmojiParameter.js]: don't fucking judge me

Comment (pedrofracassi) [reply src/structures/command/parameters/types/EmojiParameter.js]: looool

Comment (metehus) [reply src/structures/command/parameters/types/EmojiParameter.js]: me 1am

Comment (metehus) [reply src/structures/command/parameters/types/EmojiParameter.js]: wait is this right or not?

Comment (metehus) [reply src/structures/command/parameters/types/EmojiParameter.js]: ok i think not

Comment (davipatury) [reply src/structures/command/parameters/types/EmojiParameter.js]: it's

**Pull request merged**